### PR TITLE
fix(ARCH-631): use css module filename

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

our styles use css module but the filename does not reflect it.
This block us to use create-react-app kind of tools

**What is the chosen solution to this problem?**

* create a script to parse js, ts, tsx and rename found scss files.
* apply script on the repository

The script is available on gist: https://gist.github.com/jmfrancois/402c32c22fba98f1e35599f1e0dab2c2

**Please check if the PR fulfills these requirements**

- [x] The PR have used `yarn changeset` to a request a release from the CI if wanted.
- [ ] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: toscssModule.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
```
Comment (Benrajalu) [inline toscssModule.js:1]: Are you sure you want to commit this?

**Review by Benrajalu — APPROVED** [review]
`tocssModule.js` seems like it shouldn't be pushed but otherwise OK

File: toscssModule.js (hunk 1 above)
Comment (jmfrancois) [reply toscssModule.js:1]: not sure may be just put the script on a gist would be enough

**Comment by sgendre** [comment]
what do you think about embedding in the PR the script that did the work to share knowledge?

**Review by Kporal — APPROVED** [review]

**Review by tlnd-mhuchet — APPROVED** [review]

**Review by romainseb — APPROVED** [review]

**Pull request merged**